CONTRIBUTING.rst

```diff
@@ -6,7 +6,7 @@ CLI Helpers would love your help! We appreciate your time and always give credit
 Development Setup
 -----------------
 
-Read to contribute? Here's how to set up CLI Helpers for local development.
+Ready to contribute? Here's how to set up CLI Helpers for local development.
```

Comment:
@meeuw Good point. Ok, I think I fixed it -- the test suite was running all of Python's tests because pytest was finding the virutalenv's files. I made pytest only look in CLI Helpers' test directory.

I use virtualenvwrapper and store all my virtualenvs in my `$HOME/virtualenvs` directory, so I didn't notice this problem before. Might need to update mycli's as well.